cusignal_conda_env.yml

```diff
@@ -1,25 +1,17 @@
 name: cusignal
 channels:
   - nvidia
-  - rapidsai-nightly
   - conda-forge
   - defaults
-  - pytorch
+  - numba
 dependencies:
-  - pytorch
-  - cudf
-  - cuml
-  - cugraph
   - scipy
   - numpy
-  - pandas
-  - cudatoolkit=10.0
+  - cudatoolkit>=9.2,<10.2
   - boost
-  - jupyterlab
   - matplotlib
-  - nodejs
   - numba
```

Comment:
numba now occurs twice

Reply:
One numba is in the channels and another numba is in the dependencies. The Numba Jetson TX2 install says to use: `conda install -c numba numba`.

Reply:
Ah, nice. Missed that